### PR TITLE
Revue (Independent Publisher)

### PR DESCRIPTION
**Pull request by Laskewitz**
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md. 

Revue-Test
revue

### CODE REVIEW

File: independent-publisher-connectors/Revue/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,676 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Revue",
+    "description": "Revue makes it easy for writers and publishers to send editorial newsletters \u2014 and get paid.",
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Revue/apiDefinition.swagger.json:5]: @Laskewitz Unicodes - The connector's swagger still have unicodes which may not be supported in all the Power Automate page, Can you please replace all the unicodes with respective characters in the English language.

File: independent-publisher-connectors/Revue/apiProperties.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#000000",
+    "capabilities": [],
+    "publisher": "Daniel Laskewitz"
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Revue/apiProperties.json:20]: @Laskewitz Please add "stackOwner" property.

Documentation - [Add metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3-add-metadata)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: independent-publisher-connectors/Revue/apiDefinition.swagger.json (hunk 1 above)
Comment (Laskewitz) [reply independent-publisher-connectors/Revue/apiDefinition.swagger.json]: @sriyen-msft Fixed!

File: independent-publisher-connectors/Revue/apiProperties.json (hunk 2 above)
Comment (Laskewitz) [reply independent-publisher-connectors/Revue/apiProperties.json]: @sriyen-msft Fixed!

**Comment by sriyen-msft** [comment]
@Laskewitz We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps. Thanks.

**Comment by sriyen-msft** [comment]
[[certify-connector]]

**Review by sriyen-msft — APPROVED** [review]

**Comment by sriyen-msft** [comment]
@Laskewitz Apologies for the delay in response. 

Congratulations! The deployment of your connector and documentation page is now complete, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program, Thanks.

**Pull request merged**